src/benchmark.cc

```diff
@@ -131,50 +142,41 @@ bool CpuScalingEnabled() {
   return false;
 }
 
-// Given a collection of reports, computes their mean and stddev.
-// REQUIRES: all runs in "reports" must be from the same benchmark.
 void ComputeStats(const std::vector<BenchmarkReporter::Run>& reports,
                   BenchmarkReporter::Run* mean_data,
                   BenchmarkReporter::Run* stddev_data) {
   // Accumulators.
   Stat1_d real_accumulated_time_stat;
   Stat1_d cpu_accumulated_time_stat;
-  Stat1_d items_per_second_stat;
   Stat1_d bytes_per_second_stat;
-  Stat1_d iterations_stat;
-  Stat1MinMax_d max_heapbytes_used_stat;
+  Stat1_d items_per_second_stat;
+  int64_t total_iters = 0;
 
   // Populate the accumulators.
   for (std::vector<BenchmarkReporter::Run>::const_iterator it = reports.begin();
-       it != reports.end(); ++it) {
+       it != reports.end();
+       it++) {
     CHECK_EQ(reports[0].benchmark_name, it->benchmark_name);
+    total_iters += it->iterations;
```

Comment:
The iteration count is computed so that the benchmark runs in about 1.4x min_time. Computing the iteration count is a rather expensive operation because it requires repeatedly re-running the benchmark on sample iteration counts until you find one large enough. I don't think we want to repeat this operation for every repetition.

How many benchmarks differ by at least 40% between repetitions?